lib/tags/group-settings.tag

```diff
@@ -213,6 +217,7 @@
                                 }
                             } else {
                                 self.refs.description.classList.remove('graphjs-success');
+                                self.refs.description.removeAttribute('data-savesuccesstext');
```

Comment:
Please remove this change. self.refs.description refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above